File: src/main/java/ru/mokeev/ComputerGraphics/ComputerGraphicsApplication.java
package ru.mokeev.ComputerGraphics;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class ComputerGraphicsApplication {

	public static void main(String[] args) {

		new DrawingComponent().draw();

		SpringApplication.run(ComputerGraphicsApplication.class, args);
	}

}

File: src/main/java/ru/mokeev/parent/ComputerGraphicsApplication.kt
package ru.mokeev.parent

import org.springframework.boot.Banner
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
open class ComputerGraphicsApplication

fun main(args: Array<String>) {
    runApplication<ComputerGraphicsApplication>(*args){
        setBannerMode(Banner.Mode.OFF)
        setHeadless(false)
    }
}
